src/controllers/userActivites.controller/insertUserActivity/index.js

```diff
@@ -31,15 +31,12 @@ const getUserPoints = catchAsync(async (req, res) => {
 
   const userId = getUserIdQueryResponse.user[0].id;
 
-  const insertUserActivityVariables = {
-    userId,
-    activityId,
-    direction,
-    status: 'unprocessed',
-  };
-  const insertUserActivityResponse = await Hasura(insertUserActivityQuery, insertUserActivityVariables);
+  insertUserActivity(activityIdentifier, 'positive', userId, [entityId]);
```

Comment:
Can we validate entity id? For example for share project activity what is they send a project is that doesn't exist?